defender-endpoint/mac-whatsnew.md

```diff
@@ -58,6 +58,20 @@ For more information on Microsoft Defender for Endpoint on other operating syste
 
 Behavior monitoring monitors process behavior to detect and analyze potential threats based on the behavior of the applications, daemons, and files within the system. As behavior monitoring observes how the software behaves in real-time, it can adapt quickly to new and evolving threats and block them. To learn more, see [Behavior Monitoring in Microsoft Defender for Endpoint on macOS](behavior-monitor-macos.md).
 
+### Oct-2024 (Build: 101.24092.0004  | Release version: 20.124092.4.0)
+
+| Build:             | **101.24092.0004**         |
+|--------------------|-----------------------|
+| Release version:   | **20.124092.4.0** |
+| Engine version:    | **1.1.24080.11**       |
+| Signature version: | **1.421.14.0**      |
+
+##### What's new
+
+- Bug and performance fixes
+- osqueryi crash on macos with --verbose arg
```

Comment:
I believe it was fixed in a previous release (hot-fix), so no need to mention it here? (and if you mention, change it to smth like "Fixed: osqueryi crash"